test/e2e/storage/csi_mock/csi_volume_expansion.go

```diff
@@ -150,11 +151,8 @@ var _ = utils.SIGDescribe("CSI Mock volume expansion", func() {
 				if pvcSize.Cmp(newSize) != 0 {
 					framework.Failf("error updating pvc size %q", pvc.Name)
 				}
-				if test.expectFailure {
-					err = testsuites.WaitForResizingCondition(ctx, pvc, m.cs, csiResizingConditionWait)
```

Comment:
It's probably better to use `gomega.Consistently(...).WithTimeout(csiResizingConditionWait)` than dropping the use of this constant.

Reply:
thanks reply. sure, will handle this and remaining issues as soon as possible.